lib/ae_mdw/db/stream/query/parser.ex

```diff
@@ -45,6 +45,10 @@ defmodule AeMdw.Db.Stream.Query.Parser do
     {ids, types}
   end
 
+  def parse(_query_groups) do
+    {[], []}
```

Comment:
`{Map.new(), MapSet.new()}` to match the types.

Reply:
Nice catch, done